composer.json

```diff
@@ -30,5 +30,9 @@
         "pear-pear/HTTP_Request2": "*",
         "simple-cas/simple-cas": "dev-master",
         "zaininnari/html-minifier": "*"
+    },
+    "config": {
+      "secure-http": false
```

Comment:
Well crud, I forget the the channel and releases XML still reference the non-secure urls.